cli/src/print.rs

```diff
@@ -345,19 +565,149 @@ mod tests {
+        assert_eq!(
+            print.option.to_show("colsep".into()).unwrap(),
+            r#"colsep ",""#
+        );
+
+        test!(
+            &Payload::Select {
+                labels: ["id", "title", "valid"]
+                    .into_iter()
+                    .map(ToOwned::to_owned)
+                    .collect(),
+                rows: vec![
+                    vec![
+                        Value::I64(1),
+                        Value::Str("foo".to_owned()),
+                        Value::Bool(true)
+                    ],
+                    vec![
+                        Value::I64(2),
+                        Value::Str("bar".to_owned()),
+                        Value::Bool(false)
+                    ],
+                ],
+            },
+            "
+id,title,valid
+1,foo,TRUE
+2,bar,FALSE"
+        );
+
+        // ".set colwrap '" should set column separator as "'"
+        print.set_option("colwrap".into(), "'".into()).unwrap();
+        assert_eq!(
+            print.option.to_show("colwrap".into()).unwrap(),
+            r#"colwrap "'""#
+        );
+        test!(
+            &Payload::Select {
+                labels: ["id", "title", "valid"]
+                    .into_iter()
+                    .map(ToOwned::to_owned)
+                    .collect(),
+                rows: vec![
+                    vec![
+                        Value::I64(1),
+                        Value::Str("foo".to_owned()),
+                        Value::Bool(true)
+                    ],
+                    vec![
+                        Value::I64(2),
+                        Value::Str("bar".to_owned()),
+                        Value::Bool(false)
+                    ],
+                ],
+            },
+            "
+'id','title','valid'
+'1','foo','TRUE'
+'2','bar','FALSE'"
+        );
+
+        // ".set header OFF should print without column name"
+        print.set_option("heading".into(), "OFF".into()).unwrap();
+        test!(
+            &Payload::Select {
+                labels: ["id", "title", "valid"]
+                    .into_iter()
+                    .map(ToOwned::to_owned)
+                    .collect(),
+                rows: vec![
+                    vec![
+                        Value::I64(1),
+                        Value::Str("foo".to_owned()),
+                        Value::Bool(true)
+                    ],
+                    vec![
+                        Value::I64(2),
+                        Value::Str("bar".to_owned()),
+                        Value::Bool(false)
+                    ],
+                ],
+            },
+            "
+'1','foo','TRUE'
+'2','bar','FALSE'"
+        );
+
+        // ".set tabular ON" should recover default option: colsep("|"), colwrap("")
+        print.set_option("tabular".into(), "ON".into()).unwrap();
+        assert_eq!(
+            print.option.to_show("colsep".into()).unwrap(),
+            r#"colsep "|""#
+        );
```

Comment:
I still don't understand why as soon as `.set tabular ON`, colsep became `"|"` even though i did not implement yet. 🤔